.github/workflows/stale_branches.yml

```diff
@@ -0,0 +1,20 @@
+name: Run O11y Stale Branch POC
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 */6 * * *'
```

Comment:
Let's update this to once a day, before or after working hours.

Reply:
This has been done so I'll resolve this conversation.